src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -55,58 +72,157 @@ export const addInvoiceForRecipient = async ({
   memo = "",
   descriptionHash,
 }: AddInvoiceForRecipientArgs): Promise<LnInvoice | ApplicationError> => {
-  const recipientWalletIdChecked = checkedToWalletId(recipientWalletId)
-  if (recipientWalletIdChecked instanceof Error) return recipientWalletIdChecked
+  const walletIdChecked = checkedToWalletId(recipientWalletId)
```

Comment:
so:
1 - the context of recipient is useful mainly when there is also a sender, which is not the case inside this function. 
2 - shorter name making function one liner instead of many lines, which help for the readability